src/inference/src/threading/ie_cpu_streams_executor.cpp

```diff
@@ -107,13 +112,23 @@ struct CPUStreamsExecutor::Impl {
                                 return p.second > streamId_wrapped;
                             })
                             ->first;
-                    const auto max_concurrency = selected_core_type == 0 ? _impl->_config._threads_per_stream_small
-                                                                         : _impl->_config._threads_per_stream_big;
+                    const auto max_concurrency =
+                        hybrid_core ? (selected_core_type == 0 ? _impl->_config._threads_per_stream_small
```

Comment:
I would propose to have alias for selected_core_type == 0 with meaningful name.

Reply:
Ok, done.